adapter/src/logger.ts

```diff
@@ -103,9 +121,11 @@ export class Logger {
 		this._currentLogger = new InternalLogger(logCallback, logToConsole);
 		this._logFilePathFromInit = logFilePath;
 
+		// Set timestamps to false for initialization logging
+		this._prependTimestamp = false;
```

Comment:
I wanted the timestamp to be configurable, and I'm currently doing it through the launch json. However the launch json doesn't get processed until after the initialization has already gotten logged. I was told that, since this request came from internally (not from customer), we should by default leave the timestamp off from the initial portion.

It looks like this.
![img](https://user-images.githubusercontent.com/16889261/46226876-7b48db00-c312-11e8-90c7-581c515d4d6d.png)

Do you think the timestamp should be in the initialization portion by default? If the user indicates in the launch json that they do not want timestamps, the beginning portion will still have them.

Reply:
This is fine